packages/compass-indexes/src/components/create-index-field/create-index-field.module.less

```diff
@@ -13,6 +13,9 @@
     text-transform: none;
     padding-right: 10px;
     white-space: nowrap;
+    input {
+      outline: none;
```

Comment:
Now with web defaults, this input which overlays the select in the create index modal was giving a double outline. 
We have new styles to implement for this modal component which is good since it's a bit of a misuse of some html elements which generate that double outline. This change here just ensures we don't show a double outline.